src/App.js

```diff
@@ -1,21 +1,33 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import logo from './logo.svg'
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+      <div className="container-fluid">
+        <div className="row">
+          <div className="col-4" style={{ background: "lightGray" }}>
+            <br/>
+            <div className="media">
+              <img
+                style={{ width: '64px', height: '64px', background: "gray" }}
+                className="d-flex mr-3"
+                alt="React logo"
+                src={logo}/>
+              <div className="media-body">
+                <h1 className="h5">Megalodoc</h1>
+                <em>That's a bad fish</em>
```

Comment:
Just out of curiosity is there a way to escape the ' and make the git diff happy or is that just unavoidable. Super minor, ignore me 
![disappear](https://cloud.githubusercontent.com/assets/7366727/26812938/d2f9fa3a-4a48-11e7-95c1-0aab3393c2fa.gif)